backend/go.mod

```diff
@@ -160,3 +160,5 @@ require (
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace go.vallahaye.net/connect-gateway => github.com/birdayz/connect-gateway v0.0.0-20231031164214-a6a1a5702281
```

Comment:
Add fix from @birdayz fork, this is necessary for adding support for custom headers, [PR](https://github.com/vallahaye/connect-gateway/pull/31)

Reply:
pls change to 
```
replace go.vallahaye.net/connect-gateway => github.com/birdayz/connect-gateway v0.0.0-20231105183723-cad0472a8a02
```

Reply:
The changes made by @birdayz are available in the latest version of the Connect-Gateway ([v0.3.1](https://github.com/vallahaye/connect-gateway/pull/32)). The `replace` instruction is no longer required.